examples/offline/d4rl_cql.py

```diff
@@ -202,6 +231,8 @@ def watch():
 
     if not args.watch:
         replay_buffer = load_buffer_d4rl(args.expert_data_task)
+        if args.calibrated:
```

Comment:
Should this option be part of the CQL implementation instead of the example?

Reply:
It's better to preprocess the whole buffer at once and one time, than to preprocess batches as they arrive in the CQL implementation.

Reply:
See below, you can do that in `process_fn`

Reply:
Closing, this is discussed in other threads